src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Figures {
+    SQUARE, RECTANGLE, RIGHTTRIANGLE, CIRCLE, ISOSCELESTRAPEZOID;
```

Comment:
let's remove this class
if we want to get class name. let's use getClass().getSimpleName()

Reply:
Please check the solution that I implemented inside the RandomFigure class. Could not understand how I can use getClass().getSimpleName() to get random class name